plugin.audio.kxmxpxtx.bandcamp/resources/lib/cache/storageserverdummy.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Exclude from submission. Debug files?

Reply:
No, this is required to include as per:
https://kodi.wiki/view/Add-on:Common_plugin_cache

> Remember to include a fallback storageserverdummy.py file(Copy it from the script.common.plugin.cache/lib/ directory)